### ISSUE TITLE
[경기도] 메뉴, 게시판 구성

### ISSUE THREAD

**Issue by anne-parti** [parti-coop/parti-mx-spa#117]
게시판 순서, 명칭은 변경될 가능성이 있습니다.
변경될 경우 그룹 설정에서 직접 수정하겠습니다. 

### 홈
- 기존 어울림 축제와 동일한 구성

---
### 도미니를 소개합니다 
정적 페이지
- 컨텐츠 @경기도
    - 사이트 소개
    - 각 게시판 소개
    - 참여 유도 
- 디자인 @빠띠

---
### 소식/ 공지
게시판 타입: 소식

---
### 삼삼오오 신청
게시판 타입: 제안
- 고정 태그
    - 작성 페이지에 고정 태그 선택 추가 
        - 입력 항목 타이틀, 상세 내용 추후 전달
    - 게시판 목록 상단에 고정 태그 목록 노출
- 지도
    - 주소 입력(작성), 지도로 보기(글목록) 제공 

---
### 정책상상 신청
게시판 타입: 제안
- 고정 태그
    - 작성 페이지에 고정 태그 선택 추가 
        - 입력 항목 타이틀, 상세 내용 추후 전달
    - 게시판 목록 상단에 고정 태그 목록 노출
- 지도
    - 주소 입력(작성), 지도로 보기(글목록) 제공 

---
### 활동계획 
게시판 타입: 소식

---
### 투표
게시판 타입: 투표

---
### 모임/행사 공지
게시판 타입: 모임

---
### 활동기록/후기
게시판 타입: 소식
- 활동 타입 구분: 정책상상 / 삼삼오오
    - 작성 페이지에서 라디오 버튼으로 선택
         - 선택한 내용은 태그로 추가
     -  게시판 목록 상단에 고정 태그와 동일한 형식으로 노출, 태그 목록 중 가장 앞쪽에 위치
- 고정 태그
    - 작성 페이지에 고정 태그 선택 추가 
        - 입력 항목 타이틀, 상세 내용 추후 전달
    - 게시판 목록 상단에 고정 태그 목록 노출
- 지도
    - 주소 입력(작성), 지도로 보기(글목록) 제공

**Comment by anne-parti** [comment]
기본 입력항목 외 수정 필요한 내용은 https://github.com/parti-coop/parti-mx-spa/issues/124 이 이슈에서 관리합니다.

**Comment by anne-parti** [comment]
@partisoy 

현재 어울림축제 성과공유, 공지 게시판(타입: 소식) 목록은 어울림축제용으로 따로 디자인 된 것인데요. - 썸네일 크게, 제목 하단에 태그 노출
1. 도미니에서 사용할 소식 타입 게시판인 소식/공지, 활동계획, 활동기록/후기 게시판은 목록을 어떤 식으로 보여줄 건지 알려주세요. 
2. 게시글 이전할 게시판이 맞는지도 다시 한 번 확인 부탁드립니다. 


**소식 게시판 타입**
1. 현재 어울림축제 용으로 그대로 사용
    - 썸네일 있음
    - 제목 하단에 태그 노출
2. 변경된 소식 게시판 사용 (녹전연)
    1. 게시판 타입 예시: https://greenduck.kr/test/0qFk8egKBARikI6iVWg0
    2. 미리보기 타입 예시: https://greenduck.kr/test/UUmh71BdLzzjuNC83kwW
    3. 앨범형 타입 예시
        - 기본: https://greenduck.kr/test/G8DTEjRyWFZI5xQAzuL2  
        - 공지 지정 시: https://greenduck.kr/test/c6CVYEHBv3cunMT0vghe

**게시글 이전**
- 어울림 성과공유 → 도미니 활동기록/후기
- 어울림 이벤트/공지 → 도미니 소식/공지

**Comment by partisoy** [comment]
1. 각 게시판별 목록은 아래처럼 보여주시면 됩니다.
- 소식/공지 게시판과 활동계획 게시판은 ['미리보기 타입'](https://greenduck.kr/test/UUmh71BdLzzjuNC83kwW)으로 해주시면 됩니다.
- 활동기록 후기 게시판은 어울림축제 기존 형식으로 해주시고요.

2. 게시글 이전 관련 내용은 맞습니다.

**Comment by anne-parti** [comment]
> @anne-parti
> 
> 1. 각 게시판별 목록은 아래처럼 보여주시면 됩니다.
> 
> * 소식/공지 게시판과 활동계획 게시판은 ['미리보기 타입'](https://greenduck.kr/test/UUmh71BdLzzjuNC83kwW)으로 해주시면 됩니다.
> * 활동기록 후기 게시판은 어울림축제 기존 형식으로 해주시고요.
> 
> 1. 게시글 이전 관련 내용은 적어주신 게 맞습니다.

@jenna1k 소식 게시판 타입 이렇게 적용 가능할까요?

**Comment by jenna1k** [comment]
> @jenna1k 소식 게시판 타입 이렇게 적용 가능할까요?
 네, 그럼 소식게시판 보기 타입 선택 옵션없이 모두 미리보기 타입으로 바꾸고 후기게시판만 기존 형식으로 남기겠습니다.
